carl_navigation/Obstacle[] localizationObstacles #list of obstacles for the localization map layer
carl_navigation/Obstacle[] navigationObstacles   #list of obstacles for the navigation map layer